src/app/modules/gb-data-selection-module/table-components/gb-table-dimensions/gb-table-dimensions.component.ts

```diff
@@ -9,15 +9,47 @@ import {TableService} from '../../../../services/table.service';
 })
 export class GbTableDimensionsComponent implements OnInit {
 
-  public previouslySelectedDimensions: Dimension[];
-  public selectedDimensions: Dimension[];
-
   constructor(private tableService: TableService) {
   }
 
   ngOnInit() {
-    this.previouslySelectedDimensions = [];
-    this.selectedDimensions = [];
+  }
+
+  updateTable() {
+    this.tableService.mockDataUpdate();
+  }
+
+  /**
+   * This function handles the drop event when the user is reordering dimensions within
+   * the same row-dimension container or the same column-dimension container
+   */
+  onDrop() {
```

Comment:
why is this needed? is the [dragdrop]=true option not working correctly?

Reply:
primeng does not provide this event, when the user drags to reorder the dimensions, nothing happens! typical

Reply:
hmm, the demo on their website seemed working, but then probably they broke something. Ok, so please ignore my comment here